src/finn/custom_op/fpgadataflow/iodma.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+from onnx import TensorProto, helper
+from finn.core.datatype import DataType
+from finn.custom_op.fpgadataflow import HLSCustomOp
+
+
```

Comment:
what are the assumptions about i/o shape, alignment etc for the DMA? any special considerations needed?

Reply:
Added feature documentation in comments, and the copyright header